FISH-8387 Diagnostics Tool documentation (#422)
payara/Payara-Documentation: enterprise/docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Diagnostics Tool.adoc

[[diagnostics-tool]]
= Diagnostics Tool

A tool that collects the following information from the specified target.

* Domain.xml
* Server Log
* Thread Dump
* Heap Dump
* JVM Report

Collection can be targeted towards specific a specific Instance, Deployment Group or Cluster. By default, everything is collect from the DAS, each Instance, Deployment Group and Cluster.

If the target is offline, only the Domain.xml and Server Log will be collected.

NOTE: If you are not using the default _domain1_, you must specify that domain to collect diagnostics from.

[[instance-collection]]
== Instance Collection

> Diagnostics can be collected directly from an instance when the DAS is not accessible by specifying the Asadmin port.

Reviewer: ```suggestion
Diagnostics can be collected directly from an instance if not local to the DAS by specifying the admin port of the target instance.
```
